hack/e2e/fixtures/seed.yaml

```diff
@@ -116,23 +117,24 @@ spec:
       node: { }
       spec:
         anexia:
-          location_id: b164595577114876af7662092da89f76
+          locationID: b164595577114876af7662092da89f76
     syseleven-dbl1:
       country: DE
       location: Syseleven - dbl1
       spec:
         openstack:
-          auth_url: https://api.cbk.cloud.syseleven.net:5000/v3
-          availability_zone: dbl1
-          dns_servers:
+          authURL: https://api.cbk.cloud.syseleven.net:5000/v3
+          availabilityZone: dbl1
+          dnsServers:
           - 37.123.105.116
           - 37.123.105.117
-          enforce_floating_ip: true
-          ignore_volume_az: false
+          enforceFloatingIP: true
+          ignoreVolumeAZ: false
```

Comment:
I can see these values are in https://github.com/kubermatic/dashboard/blob/master/cypress/fixtures/datacenters.json too.